install-oracle.sh

```diff
@@ -1123,42 +1123,59 @@ echo -e "Running with parameters from command line or environment variables:\n"
 set | grep -E '^(ORA_|BACKUP_|GCS_|ARCHIVE_|INSTANCE_|PB_|ANSIBLE_|CLUSTER|PRIMARY)' | grep -v '_PARAM='
 echo
 
-ANSIBLE_PARAMS="-i ${INVENTORY_FILE} ${ANSIBLE_PARAMS}"
-ANSIBLE_EXTRA_PARAMS="${*}"
+export ANSIBLE_NOCOWS=1
+
+ANSIBLE_PLAYBOOK="ansible-playbook"
```

Comment:
We call `ansible-playbook` from the other shell scripts too;  should we validate it there?
Or conversely, is there any point adding this special check for `ansible-playbook` when we don't do it for, say, `mkdir`